fourth review/swagger.yaml

```diff
@@ -0,0 +1,1366 @@
+        type: string
+        description: An optional additional field for the street address of the billing address.
+      city:
+        type: string
+        description: The city of the billing address.
+        
+      company:
+        type: string
+        description: The company of the person associated with the billing address.  
+        
+      country:
+        type: string
+        description: The name of the country of the billing address.
+      
+      country_code:
+        type: string
+        description: The two-letter code (ISO 3166-1 alpha-2 two-letter country code) for the country of the billing address.
+      
+      first_name:
+        type: string
+        description: The first name of the person associated with the payment method.
+      last_name:
+        type: string
+        description: The last name of the person associated with the payment method.
+      latitude:
+        type: string
+        format: latitude
+        description: The latitude of the billing address.
+      longitude:
+        type: string
+        format: longitude
+        description: The longitude of the billing address. 
+      name:
+        type: string
+        description: The full name of the person associated with the payment method.
+      phone:
+        type: string
+        description: The phone number at the billing address.
+      province:
+        type: string
+        description: The name of the state or province of the billing address.
+      province_code:
+        type: string
+        description: The two-letter abbreviation of the state or province of the billing address.
+      zip:
+        type: string
+        format: zip
+        description: The zip or postal code of the billing address.
+        
+        
+      
+
+  Error:
+    type: object
+    description: |  
+      Object describing errors (values) on specific attributes (keys) that prevented a draft order from being saved.
+      
+      **Example:**
+      
+      ```
+      # Response
+      HTTP/1.1 422 Unprocessable Entity
+      {
+        "errors": {
+          "shipping_line": [
+            "is invalid"
+          ],
+          "line_items": [
+            "cannot be empty"
+          ]
+        }
+      }
+      
+      ```
+            
+      
+
+  Integer:
+    properties:
+      integer:
+        type: integer
+        format: int64
+        
+
+
+parameters:
+  fields:
+    name: fields
+    in: query
+    description: Comma-separated list of fields to include in the response.
+    required: false
+    type: string
+
+
+  limit:
+    name: limit
+    in: query
+    required: false
+    description: Amount of results (default 50) (maximum 250).
+    type: integer
+    format: int64
+  page:
+    name: page
+    in: query
+    type: integer
+    format: int64
+    required: false
+    description: Page to show.
+
+    
+  since_id:
+    name: since_id
+    in: query
+    description: Restrict results to be after given ID.
```

Comment:
noting that we're using "ID" here, and `id` in the responses -- either way, if it makes sense to use the same formatting for these various "ids", then let's do it 👍